src/mailtest.rs

```diff
@@ -0,0 +1,27 @@
+extern crate cronjob;
+
+use cronjob::CronJob;
+use lettre::SmtpTransport;
+
+use crate::metrics::MAIL_TESTS;
+
+pub fn init_mailtest(mailer: SmtpTransport) {
+    let mut cron = CronJob::new("mailtest", test_mail(mailer));
+    cron.minutes("*");
+    cron.seconds("0");
+    tracing::info!("starting mailtester");
+    CronJob::start_job_threaded(cron);
+}
+
+fn test_mail(mailer: SmtpTransport) -> impl Fn(&str) -> () {
+    return move |_name: &str| match mailer.clone().test_connection() {
```

Comment:
Tror problemet vi hadde med TLS (ehlo meldingen) blir ikke avdekket ved bruk av test connecion. Så tror vi må sende en minst en EHLO melding eller test mail.

Reply:
``test_connection()` tests the connection by using the SMTP NOOP command.`, så det kan være tilstrekkelig. Ønsker å starte med dette og se om det funker.